space/authz/authz.go

```diff
@@ -113,21 +113,25 @@ func (s *KeyclaokAuthzService) Authorize(ctx context.Context, entitlementEndpoin
 	}
 	claims := tokenWithClaims.Claims.(*TokenPayload)
 
-	// RPT tokens disabled. See https://github.com/almighty/almighty-core/issues/1177
+	if claims.Authorization == nil {
+		// No autrhorization in the token. This is not a RPT token. This is an access token.
```

Comment:
By the way, the service is `KeyclaokAuthzService` instead of `KeycloakAuthzService` :)